packages/shared/src/schemas/user.ts

```diff
@@ -21,7 +22,7 @@ export const CreateUserRequestSchema = z.object({
   studentUpi: z.string().nullable().optional(),
   university: z.nativeEnum(University).nullable().optional(),
   remainingSessions: z.number().nullable().optional(),
-  image: z.string().nullable().optional(),
+  image: z.union([z.string(), z.null(), MediaSchema]),
 }) satisfies z.ZodType<CreateUserData>
 
```

Comment:
please use this instead:

```suggestion
  image: z.union([z.string(), MediaSchema]).nullable(),
}) satisfies z.ZodType<CreateUserData>

```